Core/src/Iam/PolicyBuilder.php

```diff
@@ -112,9 +151,12 @@ public function setBindings(array $bindings = [])
      * @param  array  $members An array of members to assign to the binding
      * @return PolicyBuilder
      * @throws InvalidArgumentException
+     * @throws InvalidOperationException if the policy's version is greater than 1.
```

Comment:
Would we also be able to add `@deprecated` annotations to this and `removeBindings` please? :)